projects/spectator/src/lib/mock.ts

```diff
@@ -3,10 +3,12 @@ import { FactoryProvider, Type, AbstractType } from '@angular/core';
 
 type Writable<T> = { -readonly [P in keyof T]: T[P] };
 
+export declare type UnknownFunction = (...args: Array<unknown>) => unknown & Function;
```

Comment:
The original checks used `Function` which is why I included it in the type definition, but I'm not sure if it's needed.